### PR TITLE
Various java updates.

### PR DESCRIPTION
**Pull request by tylerbenson**
### What does this PR do?
Updates the java tracing documentation for the latest release.

### Preview link
Replace the branch name and add the complete path: -->
https://docs-staging.datadoghq.com/tyler/java-docs/tracing/setup/java/
https://docs-staging.datadoghq.com/tyler/java-docs/tracing/advanced/connect_logs_and_traces/
https://docs-staging.datadoghq.com/tyler/java-docs/tracing/advanced/manual_instrumentation/

### CODE REVIEW

File: content/en/tracing/advanced/connect_logs_and_traces.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ Your language level logs *must* be turned into Datadog attributes in order for t
 {{< tabs >}}
 {{% tab "Java" %}}
 
-Enable injection in the Java Tracer's [configuration][1] by setting `Ddd.logs.injection=true` or through environment variable `DD_LOGS_INJECTION=true`.
+Enable injection in the Java Tracer's [configuration][1] by adding `-Ddd.logs.injection=true` as a jvm startup argument or through environment variable `DD_LOGS_INJECTION=true`.
```
Comment (ruthnaebeck) [inline content/en/tracing/advanced/connect_logs_and_traces.md]:
```suggestion
Enable injection in the Java tracer's [configuration][1] by adding `-Ddd.logs.injection=true` as a jvm startup argument or through environment variable `DD_LOGS_INJECTION=true`.
```

Comment (ruthnaebeck) [reply content/en/tracing/advanced/connect_logs_and_traces.md]: Should there be a capital `D` in `Ddd.logs.injection`?

File: content/en/tracing/setup/java.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Beta integrations are disabled by default but can be enabled individually.
 | Java Servlet Compatible      | 2.3+, 3.0+ | Fully Supported | `servlet`, `servlet-2`, `servlet-3`            |
 | Jax-RS Annotations           | JSR311-API | Fully Supported | `jax-rs`, `jaxrs`, `jax-rs-annotations`        |
 | Jetty (non-Servlet)          | 8+         | [Beta][7]       | `jetty`, `jetty-8`                             |
-| Netty Http Server and Client | 4.0+       | Fully Supported | `netty`, `netty-4.0`, `netty-4.1`              |
+| Netty Http Server            | 4.0+       | Fully Supported | `netty`, `netty-4.0`, `netty-4.1`              |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/java.md]:
```suggestion
| Netty HTTP Server            | 4.0+       | Fully Supported | `netty`, `netty-4.0`, `netty-4.1`              |
```

File: content/en/tracing/setup/java.md [hunk 3]
```diff
@@ -101,14 +101,18 @@ Don't see your desired web frameworks? Datadog is continually adding additional
 | Apache HTTP Client       | 4.0+        | Fully Supported | `httpclient`                                   |
 | Apache HTTP Async Client | 4.0+        | Fully Supported | `httpasyncclient`, apache-httpasyncclient      |
 | AWS Java SDK             | 1.11+, 2.2+ | Fully Supported | `aws-sdk`                                      |
+| Google Http Client       | 1.19.0+     | Fully Supported | `google-http-client`                           |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/java.md]:
```suggestion
| Google HTTP Client       | 1.19.0+     | Fully Supported | `google-http-client`                           |
```

File: content/en/tracing/setup/java.md [hunk 4]
```diff
@@ -101,14 +101,18 @@ Don't see your desired web frameworks? Datadog is continually adding additional
...
 | gRPC                     | 1.5+        | Fully Supported | `grpc`, `grpc-client`, `grpc-server`           |
 | HttpURLConnection        | all         | Fully Supported | `httpurlconnection`, `urlconnection`           |
 | Kafka-Clients            | 0.11+       | Fully Supported | `kafka`                                        |
 | Kafka-Streams            | 0.11+       | Fully Supported | `kafka`, `kafka-streams`                       |
 | Jax RS Clients           | 2.0+        | Fully Supported | `jax-rs`, `jaxrs`, `jax-rs-client`             |
+| Jersey Client            | 1.9+        | Fully Supported | `jax-rs`, `jaxrs`, `jax-rs-client`             |
 | JMS                      | 1 and 2     | Fully Supported | `jms`                                          |
-| Rabbit AMQP              | 2.7+        | Fully Supported | `amqp`, `rabbitmq`                             |
+| Netty Http Client        | 4.0+        | Fully Supported | `netty`, `netty-4.0`, `netty-4.1`              |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/java.md]:
```suggestion
| Netty HTTP Client        | 4.0+        | Fully Supported | `netty`, `netty-4.0`, `netty-4.1`              |
```

File: content/en/tracing/advanced/connect_logs_and_traces.md (hunk 1 above)
Comment (tylerbenson) [reply content/en/tracing/advanced/connect_logs_and_traces.md]: It should either be `-Ddd.logs.injection=true` which identifies the whole setting, or just `dd.logs.injection=true`.  the `-D` is part of the java command line flag signifying a system property.

**Pull request merged**